frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/CommandPaletteOptions/CommandOptions.tsx

```diff
@@ -73,7 +74,7 @@ export const CommandPaletteCommandOptions: FC = () => {
       >
         <PanelTop className={styles.itemIcon} />
         <span className={styles.itemText}>Show All Fields</span>
-        <span className={styles.keyIcon}>⌘</span>
+        <span className={styles.keyIcon}>⇧</span>
```

Comment:
The right commands are ⇧2, 3, 4 😓 
ref: https://github.com/liam-hq/liam/blob/a683c9375e19464e9d614be7d2b5c4a5bb50b70f/frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/hooks/useSubscribeCommands/useSubscribeSwitchShowMode.ts#L7-L19